manubot/cite/doi.py

```diff
@@ -210,7 +231,7 @@ def get_doi_csl_item(doi: str):
 
 
 doi_retrievers = [
-    get_doi_csl_item_datacite,
-    get_doi_csl_item_crosscite,
+    get_doi_csl_item_default,
+    get_doi_csl_item_url,
```

Comment:
Do you expect the URL-based retrieval to be more accurate than Zotero's DOI support?